client/imports/ui/pages/dashboard/dashboard.ts

```diff
@@ -54,7 +54,7 @@ export default class Dashboard extends MeteorComponent {
     mdIconRegistry.setDefaultFontSetClass('tuxicon');
 
 		this.subscribe('user-courses', () => {
-			this.courses = courses.find().fetch();
+			this.courses = courses.find({}, { limit: 5 }).fetch();
```

Comment:
Limit to display 5 enrolled courses on the main dashboard, click more button to go to CourseList which displays all enrolled courses.